components/modal/Modal.tsx

```diff
@@ -188,7 +194,8 @@ export default class Modal extends React.Component<ModalProps, {}> {
 
     return (
       <Dialog
-        {...this.props}
+        {...restProps}
```

Comment:
prefixCls is missing in restProps

Reply:
<img width="779" alt="image" src="https://user-images.githubusercontent.com/507615/43451897-98a71002-94e8-11e8-9307-42d550557f85.png">

Reply:
Please run `npm test` in your local enviorment to make sure test is OK.

Reply:
You can keep `this.props` usage.

Reply:
I want to drop the `centered`.

Reply:
👌